services/showcase/api/endpoints/routes/line_of_business_issuer.py

```diff
@@ -50,7 +50,8 @@ async def issue_degree(
     # we will need some info for revoking this (rev_reg_id, cred_rev_id)
```

Comment:
i didn't think we were changing showcase... v0 was supposed to remain intact as is.
i guess this doesn't add or remove anything functionally, but we need to be careful here - was this just put in for debug purposes? if so, can you remove it, otherwise we have to redeploy showcase everywhere.

Reply:
see comment below, fixing a showcase bug.